README.md

```diff
@@ -122,6 +122,12 @@ Then call `subscribe` to listen to location change events.
 const unsubscribe = location.subscribe(main.location)
 ```
 
+### State First
+
+By default, for convenience, the router use the `window.location` property as source of truth.
+If you want to ensure the router to use the state as only source of truth : you are responsible of wiring the `location` slice to your `state` and `actions`.
+Then call `subscribe` to listen to location change events and reflect it into the `state`.
```

Comment:
So, by default your app would not react to `pushState` (i.e. clicks on `Link` components) because no state change is triggered? So what actually changes 🤔

Reply:
`pushState` will change `window.location` so at the next render, the view will follow the right route. But yes mostly. 
Maybe this will be fixed when I will introduce Router Lifecycle Events.